.azure-pipelines/linux-conda-CI.yml

```diff
@@ -70,6 +70,10 @@ jobs:
       pip install flake8
     displayName: 'install flake8'
 
+  - script: |
```

Comment:
https://github.com/onnx/sklearn-onnx/blob/fd48cc12a2485f46ef4370e1473419fb219d21a8/.azure-pipelines/win32-conda-CI.yml#L52-L54

Reply:
Thanks for the PR, @c-bata! We just discussed this in #99 and decided to remove the install since flake8 already runs in the windows build.

cc: @xadupre

Reply:
I see. I close this.